x-pack/test/detection_engine_api_integration/security_and_spaces/tests/generating_signals.ts

```diff
@@ -1265,7 +1265,7 @@ export default ({ getService }: FtrProviderContext) => {
           parents: [
             {
               rule: signalNoRule.parents[0].rule, // rule id is always changing so skip testing it
-              id: 'b63bcc90b9393f94899991397a3c2df2f3f5c6ebf56440434500f1e1419df7c9',
+              id: signalNoRule.parents[0].id,
```

Comment:
I wasn't exactly sure why this started braking after pulling in master - noticing the comment a line above, I used the same strategy, please confirm if this is an appropriate fix

Reply:
@ecezalp Did the ID change because the `_id` calculation in `wrap_hits_factory` changed to include the version? I suspect that's the case. If so, we probably want to hard-code the new calculated id here, as it should be the same every time.

Reply:
uhh that makes sense. I updated the ids. locally I got a 409 error which I thought was strange considering that I was just updating the comparison object used in the assertion, I pushed the change anyway, hopefully it should work as expected.